### DISCUSSION TITLE
error while installing ImageMagick version 7.1.1

### DISCUSSION THREAD

**Discussion by SREESHMARAJ659** [ImageMagick/ImageMagick#6394] [Help]
### ImageMagick version

7.1.1

### Operating system, version and so on

Ubuntu 22.04.2 LTS

### Description

Hello, 
i downloaded the tar.gz file of above mentioned version of ImageMagick in my system. build was completed and while running make i got the following error.

make  all-am
make[1]: Entering directory '/home/ibsdshl/ImageMagick-7.1.1'
  CXXLD    utilities/magick
/usr/bin/ld: MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `zip_fopen'
/usr/bin/ld: MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `zip_fread'
/usr/bin/ld: MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `zip_discard'
/usr/bin/ld: MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `zip_open'
/usr/bin/ld: MagickCore/.libs/libMagickCore-7.Q16HDRI.so: undefined reference to `zip_fclose'
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:8188: utilities/magick] Error 1
make[1]: Leaving directory '/home/ibsdshl/ImageMagick-7.1.1'
make: *** [Makefile:6236: all] Error 2
 please help me to fix this error.
Thank you in advance

**Comment by urban-warrior** [comment]
Type

```
$ ./configure
```

and check the output for LIBS near the very end of the output.  We get `-lz` near the end of the LIBS string.  For example, we get `-lxml2 -lz -lm -lpthread`.  Do you?  The Z library satisfies the API references you received.
